### DISCUSSION TITLE
Git sync from multiple repositories

### DISCUSSION THREAD

**Discussion by pollynesterovich** [apache/airflow#23366] [General]
Hi, i would like to implement multi-repository support for dags using helm chart. I set up submodules that link to 2 repositories. But the problem is that they are not updated inside the airflow-scheduler. Can you please tell me how to implement this? Normally I would use the command `git submodule update --init --remote --recursive`

**Comment by potiuk** [comment]
Look at git-sync docs. It has options to support submodules. Airflow uses just standard git-sync container,  you should look there.

**Reply by pollynesterovich** [reply]
@potiuk I add to values.yaml next:
`env: 
      - name: "GIT_SYNC_SUBMODULES"
        value: "recursive"`
but the contents of the submodules are not updated.  Please tell me how can i do this?

**Reply by potiuk** [reply]
I think you should ask in git-sync forums. I have no idea really without debugging it myself.

**Reply by potiuk** [reply]
Just check if the env is correctly setup, if you see it in container - and pass all the information as issue in git-sync forum, if you cannot get it to work :)

**Reply by pollynesterovich** [reply]
Ok, thanks.
